rosdep/python.yaml

```diff
@@ -2912,9 +2912,6 @@ python-psycopg2:
   debian: [python-psycopg2]
   fedora: [python-psycopg2]
   gentoo: [=dev-python/psycopg-2*]
-  ubuntu:
-    '*': [python-psycopg2]
-    focal: [python3-psycopg2]
```

Comment:
I'm wondering if this is really right?
`ubuntu: [python-psycopg2]`
Would not work for foxy and jammy, but maybe this would be needed for some legacy distros?

Reply:
> Would not work for foxy and jammy, but maybe this would be needed for some legacy distros?

We don't typically worry about distributions that are end-of-life (though we also don't actively remove those keys either).  So in this case, I think it is OK.